src/user/screens/follower-list.screen.js

```diff
@@ -56,13 +75,15 @@ class FollowerList extends Component {
                   navigation={navigation}
                   showFullName
                 />}
+              onEndReachedThreshold={0.5}
+              onEndReached={() => getFollowers(login, true)}
```

Comment:
this is how pagination is done :) Passing true as the second parameter for the data fetcher